src/Cli/dotnet/CommonArguments.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using Microsoft.DotNet.Cli.Utils;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal class CommonArguments
+    {
+        #region PackageIdentityArgument
```

Comment:
I don't have a problem with this, personally, but I've been told that regions are "out-of-style" or something like that.